README.md

```diff
@@ -1,36 +1,41 @@
-    * ***`AUTH_KEY`***: Use your maker authorization key here. This is the same key you provide when connecting to our maker WebSocket.
-      * Wallets: In the current iteration, no funds are transferred. This is just used to fill in the `trader` field in the RFQs you get.
-        * ***`QA_TAKER_ADDRESS`***: Put the EVM address of the wallet you want to use for testing. 
-        * ***`QA_TAKER_ADDRESS_SOLANA`*** Put the Solana address of the wallet you want to use for testing. 
+   - **_`AUTH_KEY`_**: Use your maker authorization key here. This is the same key you provide when connecting to our maker WebSocket.
+     - Wallets: In the current iteration, no funds are transferred. This is just used to fill in the `trader` field in the RFQs you get.
+       - **_`QA_TAKER_ADDRESS`_**: Put the EVM address of the wallet you want to use for testing.
+       - **_`QA_TAKER_ADDRESS_SOLANA`_** Put the Solana address of the wallet you want to use for testing.
 4. Finally, run `yarn qa <options>` with the following options:
-    * ***`--maker`***: your external maker name (e.g. `--maker=mm123`)
-    * ***`--chain`***: chain id that you want to test (e.g. `--chain=137`)
-    * ***(Optional) `--chain_type`***: chain type to test (either `evm` or `solana`. Default: `evm`)
-    * ***(Optional) `--quote_chain`***: quote chain to test. If not set, we will use the base_chain.
-    * ***(Optional) `--quote_chain_type`***: quote chain type to test. (Default: 'evm')
-    * ***(Optional) `--check_all_xchain`***: if true, request quotes for all intra-chain and cross-chain quotes available for the provided base chain. (Default: `false`)
-    * ***(Optional) `--base_token`***: base token name if you want to only test a specific pair (e.g. `--base_token=ETH`). If this is set, you also need to provide `--quote_token`. If this is not set, we'll test all pairs on the provided `chain`.
-    * ***(Optional) `--quote_token`***: quote token name if you want to only test a specific pair (e.g. `--quote_token=ETH`). If this is set, you also need to provide `--base_token`. If this is not set, we'll test all pairs on the provided `chain`.
-    * ***(Optional) `--env`***: Environment to test against. Default is `--env=staging`, the other option is `--env=production`. This will route your requests against `api-staging.hashflow.com` or `api.hashflow.com` respectively.
-    * ***(Optional) `--num_requests`***: Number of RFQs to send for each pair. Default is `--num_requests=30`.
-    * ***(Optional) `--delay_ms`***: Delay to introduce between each RFQ in milliseconds. This can be useful to test pricing skew over a longer period of time. Default is `--delay_ms=0`.
-    
+   - **_`--maker`_**: your external maker name (e.g. `--maker=mm123`)
+   - **_`--chain`_**: chain id that you want to test (e.g. `--chain=137`)
+   - **_(Optional) `--chain_type`_**: chain type to test (either `evm` or `solana`. Default: `evm`)
```

Comment:
Hmm, prettier made these changes. I'm not sure if they're all reasonable.

Reply:
@geoffrey-hashflow I think we should disable prettier for this file.  There are two tools that we can leverage to tune prettier:
- prettierrc -- this can tell prettier which file formats to affect (ideally we only set it to `ts`)
- prettierignore -- this tells prettier which files to ignore; typically, we ignore node_modules and auto-generated files

Reply:
Reverted this file. Added rule to ignore *.md. Formatting json and ts seems reasonable.